src/gatsby/src/pages/plan-your-trip/park-operating-dates.js

```diff
@@ -54,6 +59,42 @@ const ParkLink = ({ park, advisories }) => {
     }
   }
 
+  // get advisories
+  const getAccessStatus = (status) => {
+    setParkAccessStatus(status)
+  }
```

Comment:
Should this name be "set" instead of "get" ? And if so, couldn't you just call setParkAccessStatus without this other function?